planemo/network_util.py

```diff
@@ -33,10 +33,11 @@ def wait_http_service(url, timeout=None):
 
 # code.activestate.com/recipes/576655-wait-for-network-service-to-appear
 def wait_net_service(server, port, timeout=None):
-    """ Wait for network service to appear
-        @param timeout: in seconds, if None or 0 wait forever
-        @return: True of False, if timeout is None may return only True or
-                 throw unhandled network exception
+    """ Wait for network service to appear.
+
+    @param timeout: in seconds, if None or 0 wait forever
+    @return: True of False, if ``timeout`` is ``None`` may return only True or
+    throw unhandled network exception
```

Comment:
Shouldn't that be indented?

Reply:
I tried the old way, I tried two spaces, 4 spaces, Sphinx hates everything but this.

Reply:
Typo in the line above?

``True of False`` --> ``True or False``

Reply:
Shouldn't these start with a `:` http://www.sphinx-doc.org/en/stable/domains.html#info-field-lists
```
:param timeout: ...
:return: ...
```

Reply:
Revised the docs to implement this I think.